donate/templates/pages/base_page.html

```diff
@@ -47,7 +47,8 @@
             </div>
 
             <a class="header__logo-link" href="/">
-                <img class="header__logo" src="/_images/mozilla-logo-white.svg" alt="Mozilla">
+                <img class="header__logo header__logo--mobile" src="/_images/mozilla-logo-m.svg" alt="Mozilla">
+                <img class="header__logo header__logo--desktop" src="/_images/mozilla-logo-white.svg" alt="Mozilla">
```

Comment:
should these not be referenced using the `{% static 'path/to/image' %}` tag or has it been setup differently in this codebase?

Reply:
It's set up a bit differently here because of how we're using whitenoise.

Reply:
We've updated these in the static tooling branch, will merge in